packages/web-components/src/components/cbp-card/cbp-card.scss

```diff
@@ -1,3 +1,134 @@
+:root {
+  --cbp-card-color-background: var(--cbp-color-white);
+  --cbp-card-color-border: var(--cbp-color-gray-cool-10);
+  --cbp-card-color-title: var(--cbp-color-text-darker);
+  
+  --cbp-banner-card-color: var(--cbp-color-gray-cool-60);
+  --cbp-banner-card-color-title: var(--cbp-color-text-lighter);
+  --cbp-banner-card-color-body-background: var(--cbp-color-white);
+}
+
 cbp-card {
   display: block;
+  background-color: var(--cbp-card-color-background);
+  border-radius: var(--cbp-border-radius-softer);
+  border-color: var(--cbp-card-color-border);
+  border-style: solid;
+  border-width: var(--cbp-border-size-md);
+
+  .cardbody {
+    padding: var(--cbp-space-4x);
+    width: 100%;
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  [slot=cardtitle] {
```

Comment:
This slipped my mind, but go ahead and declare these `slot` selectors as display:block as well. Since you can stick a `slot` on anything, such as a span, make sure they are `display:block` if you are treating them as a block element.